module/dkg/client.go

```diff
@@ -165,16 +159,18 @@ func (c *Client) ReadBroadcast(fromIndex uint, referenceBlock flow.Identifier) (
 // public keys for each DKG participant. Serialized pub keys are encoded as hex.
 func (c *Client) SubmitResult(groupPublicKey crypto.PublicKey, publicKeys []crypto.PublicKey) error {
 
+	started := time.Now()
 	ctx := context.Background()
```

Comment:
I think we are missing the context timeout for this method [like we have for Broadcast](https://github.com/onflow/flow-go/pull/630/files#diff-4ac05762c05d6f976ad81a2f018c10670c82ace79504268c08e0add77ad71d3cR63).